front/app/containers/Admin/projects/project/analysis/Demographics/AuthorsByAge.tsx

```diff
@@ -168,7 +168,11 @@ const AuthorsByAge = ({ customFieldId }: Props) => {
         <FormattedMessage {...messages.authorsByAge} />
       </Title>
       <ResponsiveContainer width="100%" height={100}>
-        <BarChart data={chartData} margin={{ top: 20, left: 8, right: 8 }}>
+        <BarChart
```

Comment:
We should rename our own BarChart to CustomBarChart or something, because it was confusing why you were adding accessibilityLayer here.